daemon/daemon.go

```diff
@@ -1501,6 +1501,16 @@ func (daemon *Daemon) newBaseContainer(id string) *container.Container {
 	return container.NewBaseContainer(id, daemon.containerRoot(id))
 }
 
+// Reload reads configuration changes and modifies the
+// daemon according to those changes.
+// Currently, only the labels configuration is taken into account.
```

Comment:
If we have to restart the daemon we might as well not implement this feature. Because the whole point of this is to reload without restarting. Let's see how down the :rabbit: hole I need to go to make it happen.